diff -dur ndisc6-1.0.1.orig/doc/rdnssd.8-in ndisc6-1.0.1/doc/rdnssd.8-in
--- ndisc6-1.0.1.orig/doc/rdnssd.8-in	2008-02-08 16:39:31.000000000 +0100
+++ ndisc6-1.0.1/doc/rdnssd.8-in	2011-06-15 10:12:43.000000000 +0200
@@ -63,7 +63,7 @@
 .TP
 .BR "\-u username" " or " "\-\-user username"
 Override the user that the program will run as. By default, it runs as
-.BR "nobody" "."
+.BR "rdnssd" "."
 
 .TP
 .BR "\-V" " or " "\-\-version"
--- ndisc6-1.0.8/rdnss/rdnssd.c.orig	2022-05-11 20:30:50.000000000 +0200
+++ ndisc6-1.0.8/rdnss/rdnssd.c	2025-05-15 20:46:01.549367917 +0200
@@ -757,7 +757,7 @@ version (void)
 
 int main (int argc, char *argv[])
 {
-	const char *username = "nobody", *hookpath = NULL,
+	const char *username = "rdnssd", *hookpath = NULL,
 	           *pidpath = LOCALSTATEDIR "/run/rdnssd.pid",
 	           *resolvpath = LOCALSTATEDIR "/run/rdnssd/resolv.conf";
 	int pidfd, val, pipefd = -1;
